src/screens/SettingsScreen.tsx

```diff
@@ -222,6 +223,76 @@ function FontSizePreferenceSelector() {
   );
 }
 
+function EditorFontFamilyPreferenceSelector() {
+  const dispatch = useDispatch();
+  const [selectEditorFontFamilyOpen, setEditorFontFamilyPreferenceOpen] = React.useState(false);
+  const viewSettings = useSelector((state: StoreState) => state.settings.viewSettings);
+  const { editorFontFamily } = viewSettings;
+
+  return (
+    <List.Item
+      title="Editor Font"
+      description="Set the font used in the note editor"
+      right={(props) =>
+        <FontSelector
+          {...props}
+          visible={selectEditorFontFamilyOpen}
+          selected={editorFontFamily ?? "monospace"}
+          onOpen={() => setEditorFontFamilyPreferenceOpen(true)}
+          onDismiss={() => setEditorFontFamilyPreferenceOpen(false)}
+          onChange={(selected) => {
+            setEditorFontFamilyPreferenceOpen(false);
+            if (selected == null || selected === editorFontFamily) {
+              return;
+            }
+            dispatch(setSettings({
+              viewSettings: {
+                ...viewSettings,
+                editorFontFamily: selected as typeof editorFontFamily,
```

Comment:
I'm surprised you need to do a cast here, as I think select should have the correct type. You probably need to fix the type of `FontSelector` to return the correct type here.

Reply:
I actually copied most of the code of `Select` without thinking, that must be why 😉